fiftyone/utils/ultralytics.py

```
@@ -701,6 +702,137 @@ def _predict_all(self, imgs):
+    pass
+
+
+class FiftyOneYOLOEVPModel(FiftyOneYOLOModel):
+    """YOLOE model with visual prompt (box prompt) support.
+
+    Args:
+        config: a :class:`FiftyOneYOLOEVPModelConfig`
+    """
+
+    @staticmethod
+    def collate_fn(batch):
```

Ruff (0.15.12)

[warning] Missing return type annotation for staticmethod `collate_fn`

(ANN205)